templates/LDA_Preprocessing.ipynb

```diff
@@ -1,15 +1,941 @@
 {
```

Comment:
<p>I don't understand what these plots are showing. Is this a barchart?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/468/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>It's a histogram counting the number of FOVs that fall into certain bins of average cell area, total cell counts, etc.  But it's a horrible histogram because there are only two FOVs in the example dataset.</p>

Reply:
Got it. Can you make some fake data with like 30 FOVs just to make sure the histograms look good?

Reply:
I can show you histograms from the real 67 FOVs in the PAP data if you like.  But maybe I'll Slack it to you for the sake of not uploading extra stuff?